tests/eval/test_text_structure.py

```diff
@@ -189,13 +227,22 @@ def test_contains_all_strings(generated_outputs, strings, case_sensitive,
                                       case_sensitive)
     assert eval_value.metric_name == 'contains_all_strings'
     assert eval_value.prompts is None
-    assert eval_value.generated_outputs == generated_outputs
+    if isinstance(generated_outputs, str):
```

Comment:
Would it be useful to make a util function for this pattern because it appears so frequently?

Reply:
Sure, refactoring into a function in `tests/utils.py`. Makes the tests a bit more readable in this file.